tests/elasticsearch_settings_test.py

```diff
@@ -0,0 +1,130 @@
+# -*- coding: utf-8; -*-
+#
+# This file is part of Superdesk.
+#
+# Copyright 2013, 2014 Sourcefabric z.u. and contributors.
+#
+# For the full copyright and license information, please see the
+# AUTHORS and LICENSE files distributed with this source code, or
+# at https://www.sourcefabric.org/superdesk/license
+import json
+from eve.utils import ParsedRequest
+from superdesk.tests import TestCase
+from superdesk import get_resource_service
+
+
+class ElasticSearchSettingsTest(TestCase):
+    items = [
+        {
+            '_id': '123', 'headline': 'Test 1', 'slugline': 'Soccer England', 'body_html': 'Test'
+        },
+        {
+            '_id': '456', 'headline': 'Test 2', 'slugline': 'Soccer England Result', 'body_html': 'Test'
+        },
+        {
+            '_id': '489', 'headline': 'Test 3', 'slugline': 'Soccer-England/Result', 'body_html': 'Test'
+        },
+        {
+            '_id': '321', 'headline': 'Test 4', 'slugline': 'Test Soccer England', 'body_html': 'Test'
+        },
+        {
+            '_id': '654', 'headline': 'Test 5', 'slugline': 'England Test Soccer', 'body_html': 'Test'
+        },
+        {
+            '_id': '984', 'headline': 'Test 6', 'slugline': 'Soccer Germany', 'body_html': 'Test'
+        }
+    ]
+
+    def setUp(self):
+        super().setUp()
+        with self.app.app_context():
```

Comment:
For convenience, it would probably make sense to push application context on the stack in the `setUp()` and pop it in the `tearDown()` method, avoiding the need to use the same `with` statement over and over again in every test method.